### ISSUE TITLE
The app is not minimized when "Start app on login" is enabled

### ISSUE THREAD

**Issue by gg-berdzenishvili** [mattermost/desktop#570]
I confirm (by marking "x" in the [ ] below): 

- [x] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [x] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md). 

---

**Summary**
The app is not minimized to tray on system (Linux) start-up.

**Steps to reproduce**
- Operating System: Antergos Linux x64, kernel 4.12.4-1
- Mattermost Desktop App version: 3.7.0
- Mattermost Server version: 4.0.0  (4.0.2-rc1)
- Clear steps to reproduce the issue:
  * Go to File -> Settings
  * Check "Show icon in the notification area" option just to have a tray icon
  * Check "Leave app running in notification area when application window is closed" to always have tray icon
  * _Check "Start app on login" option_
  * Reboot your system

**Expected behavior**
The app is started minimized as stated in the description to "Start app on login" option ("The app will initially start minimized and appear on the taskbar.").

Checked desktop file used for autostarting the app:
```
[Desktop Entry]
Type=Application
Version=1.0
Name=mattermost-desktop
Comment=mattermost-desktopstartup script
Exec=/usr/lib/mattermost/mattermost-desktop --hidden
StartupNotify=false
Terminal=false
```
`--hidden` option is passed

**Observed behavior**
The app is not minimized.

**Maybe useful links**
There was already some discussing regarding this topic: https://github.com/mattermost/desktop/pull/230

**Comment by gg-berdzenishvili** [comment]
Also maybe it will be good to have "Start app on login" and "Start app minimized" options separated (like in new Skype for Linux Beta). It will give user more control.

**Comment by jasonblais** [comment]
Closed via #571
